src/main.c

```diff
@@ -156,6 +157,7 @@ static void do_show_version(void)
 
 static gboolean process_cmdline(int argc, char **argv)
 {
+	setlocale(LC_ALL, "");
```

Comment:
This certainly requires a comment of sorts describing why this is necessary.
Is this documented in https://docs.gtk.org/glib/ somewhere ?

Reply:
i found some docs here:
https://docs.gtk.org/glib/i18n.html

Reply:
@coldtobi  It makes sense; calling setlocale() directly in main() provides a clearer and more straightforward implementation.